test/main_test.js

```diff
@@ -24,11 +24,12 @@ suite('End to End', () => {
     ];
     let doc = await documenter({
       schemas, // schema.id + content
+      docsFolder: path.join(__dirname, 'docs'),
```

Comment:
This won't always be in the same directory as the docs. I've used https://www.npmjs.com/package/app-root-dir for this purpose before in tc-lib-validate and like it quite a bit.

https://github.com/taskcluster/taskcluster-lib-validate/blob/master/src/validate.js#L21